plugin/aws/ec2Instance.go

```diff
@@ -217,7 +219,17 @@ func (inst *ec2Instance) Exec(ctx context.Context, cmd string, args []string, op
 		return nil, fmt.Errorf("No public interface found for %v", inst)
 	}
 
+	var identityfile string
+	if name, ok := meta["KeyName"]; ok {
+		homedir, err := os.UserHomeDir()
+		if err != nil {
+			return nil, err
+		}
+		identityfile = (filepath.Join(homedir, ".ssh", (name.(string) + ".pem")))
+	} else {
+		return nil, fmt.Errorf("No key name found for %v", inst)
```

Comment:
I still think this shouldn't fail if we can't find a key. The ssh-agent may have it.

Reply:
Ah, good catch. I think I can knock this out pretty quickly.

Reply:
Ready for review.